src/pages/resources/fundamentals/dom-versioning/index.md

```diff
@@ -0,0 +1,58 @@
+---
+keywords:
+  - InDesign DOM
+  - DOM Versions 
+  - Mount API
+---
+
+# InDesign DOM APIs
```

Comment:
@anoopvalomkot  I made some tweaks to this doc. Please give it a read and let me know if anything should be changed.

Reply:
looks good.